src/components/App/App.js

```diff
@@ -26,24 +27,20 @@ export class App extends Component {
   }
 
   componentDidMount() {
-    if (localStorage.getItem("user")) {
-      const savedUser = JSON.parse(localStorage.getItem("user"));
-      this.setState({userID: savedUser.user_id})
-      this.loadUserProjectsAndPalettes(savedUser.user_id)
-    }
   }
```

Comment:
I'm hoping to save all the necessary things to localStorage eventually, instead of just the user and then fetching their information.